server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/_footer.scss

```diff
@@ -13,74 +13,79 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@import "shared/go-variables";
 
-.footer {
-  background-color: #292c2f;
-  text-align:       center;
-  font-size:        1.2rem;
-  padding:          1rem;
+.app-footer {
+  background : $grey;
+  text-align : center;
+  font-size  : 1.2rem;
+  padding    : 1rem;
 
   .inline-list a {
-    display:       inline-block;
-    border-radius: 2px;
+    display       : inline-block;
+    border-radius : 2px;
 
     &:nth-of-type(3) {
```

Comment:
This selector was added for the green separator bar, I don't suppose we have the bar, is there any reason to keep the selector with a padding?